tests/e2e/install_test.go

```diff
@@ -52,7 +52,12 @@ var _ = Describe("E2E - Install Rancher Manager", Label("install"), func() {
 				return err
 			}, misc.SetTimeout(2*time.Minute), 5*time.Second).Should(BeNil())
 		})
-
+		if clusterType == "hardened" {
+			By("Configuring hardened cluster", func() {
+				err := exec.Command(installHardenedScript).Run()
```

Comment:
As all commands in `installHardenedScript` are run with `sudo` it might be easier to simply remove all `sudo` calls in the script and do this here:
```go
err := exec.Command("sudo", installHardenedScript).Run()
```